README.md

```diff
@@ -312,7 +312,7 @@ Remote Targets
 | Oracle Enterprise Linux      | 5, 6, 7                                          | i386, x86_64  |
 | Red Hat Enterprise Linux     | 5, 6, 7                                          | i386, x86_64  |
 | Solaris                      | 10, 11                                           | sparc, x86    |
-| Windows\*                    | 7, 8, 8.1, 10, 2008, 2008R2 , 2012, 2012R2, 2016 | x86, x86_64   |
+| Windows\*                    | 8, 8.1, 10, 2008, 2008R2, 2012, 2012R2, 2016     | x86, x86_64   |
```

Comment:
OH LOOK A TABLE! :P

Reply:
But it's on the readme and not the website! 😂